pkgs/top-level/all-packages.nix

```
@@ -8173,6 +8173,11 @@ with pkgs;
   inherit (callPackage ../development/tools/replay-io { })
     replay-io replay-node-cli;
 
+  rescript-language-server = let
+    version = "1.62.0";
+    rescript-editor-analysis = callPackage ../applications/editors/vscode/extensions/chenglou92.rescript-vscode/rescript-editor-analysis.nix { inherit version; };
+  in callPackage ../by-name/re/rescript-language-server/package.nix { inherit rescript-editor-analysis; };
```

```suggestion
  rescript-language-server = callPackage ../by-name/re/rescript-language-server/package.nix { rescript-editor-analysis = vscode-extensions.chenglou92.rescript-vscode.rescript-editor-analysis; };
```
This should be more compliant with nixpkgs-vet. It requires the change in the next comment.

---

I see, that makes sense.